helm/chaos-mesh/values.yaml

```diff
@@ -283,20 +304,20 @@ dashboard:
 
   # The keys within the "env" map are mounted as environment variables on the pod.
   env:
+    # The address which chaos-dashboard would listen on.
     LISTEN_HOST: 0.0.0.0
+    # The port which chaos-dashboard would listen on.
     LISTEN_PORT: 2333
+    # The address which metrics endpoints would listen on.
     METRIC_HOST: 0.0.0.0
+    # The ports which metrics endpoints would listen on.
     METRIC_PORT: 2334
 
     # If you'd like to use a DB other than SQLite (the default), set a driver + DSN here.
     DATABASE_DRIVER: sqlite3
+    # The db dsn used for Chaos Dashboard
     DATABASE_DATASOURCE: /data/core.sqlite
 
-    # If you are going to store build secrets in the Chaos Dashboard database, it is suggested that
-    # you set a database encryption secret. This must be set before any secrets are stored
-    # in the database.
-    # DATABASE_SECRET:
```

Comment:
I think this feature should be provided.

Reply:
I will implement it in a single pr.

Reply:
Sorry, I misunderstood the meaning of this field. At present, the data in Chaos Dashboard has no sensitive data, so this function is not necessary at present.